server/lib/config/pbench-server-default.cfg

```diff
@@ -126,6 +126,12 @@ lowerbound = 820
 # index_prefix =
 # bulk_action_count =
 
```

Comment:
It is mentioned on line 123 that `# NOTE: No defaults are provided for the "Indexing" section deliberately.` what is the reason that we don't want to include the dummy elastic endpoint in default config file that can be overwritten from the main config file? cc: @portante @ndokos 

I did this to remove the reference to actual elastic/graphql instance for unit testing, so we can just check the api workflow without actually hitting the endpoints. If this is fine I can actually reuse the Indexing config parameter instead of separate elasticsearch variable.
Any thought?

Reply:
> what is the reason that we don't want to include the dummy elastic endpoint in the default config file that can be overwritten from the main config file?

Mostly because we want to avoid providing default for services that have to be explicitly setup.

> I did this to remove the reference to actual elastic/graphql instance of unit testing ...

I think you mean that you override the configuration file in the unit tests, right?

The concern here is to make sure we try talking to services for Elasticsearch or GraphQL which don't actually exist.  DNS lookups for what we consider to be non-existent host names could actually resolve in some environments.

Let's also drop the reference in the comments to the `server` options in the `Indexing` section.  Thanks!